nim/tests/test_temp.py

```diff
@@ -0,0 +1,14 @@
+from nim.exceptions import NimException
+from nim.game import Nim
+import pytest
+
+
+@pytest.fixture
+def game_object():
+    game = Nim()
+    return game
```

Comment:
What about this?
```
def game():
    return Nim()
```
(actually asking since you know `pytest` better than I do)

Reply:
Yeah, it's much better your way. Sloppy mistake, sorry.